src/router_core/router_core_private.h

```diff
@@ -800,15 +801,7 @@ struct qdr_core_t {
     qdrc_attach_addr_lookup_t  addr_lookup_handler;
     void                      *addr_lookup_context;
 
-    //
-    // Agent section
-    //
-    qdr_query_list_t       outgoing_query_list;
-    sys_mutex_t           *query_lock;
-    qd_timer_t            *agent_timer;
-    qdr_manage_response_t  agent_response_handler;
-    qdr_subscription_t    *agent_subscription_mobile;
-    qdr_subscription_t    *agent_subscription_local;
+    qdr_agent_t               *agent;
```

Comment:
I wonder if this should be mgmt_agent instead of just agent because it is used to respond exclusively to management requests coming in on $management

Reply:
+1